demo/Dockerfile.alpine

```diff
@@ -29,6 +28,8 @@
 # base image - alpine 3.18
 FROM alpine:3.18 AS os
 
+RUN apk update
```

Comment:
Why did you move this into a separate `RUN`? 🤔

Anyway, we could also use `apk --update add` below and remove this command altogether.

Reply:
No good reason. Just to consolidate common code. I can mov it back since it will create another layer.